### ISSUE TITLE
Unable to add additional people to activities

### ISSUE THREAD

**Issue by reid** [monicahq/monica#2921]
**Describe the bug**
When adding or editing an Activity, I am unable to add an additional involved person from iOS Safari. I am able to add additional persons from desktop Firefox and desktop Safari.

**Screenshots**
Tapping on the name in the circled area has no effect.
![IMG_9DB9FA319BF9-1](https://user-images.githubusercontent.com/9517/63371304-af692800-c338-11e9-8d0d-12710760d853.jpeg)

**Which version are you using:**
 - Self-hosted Monica v2.14.0 (Docker tagged)
 - iOS 12.4, iPhone XR

**Additional context**
To reproduce:

1. Add or edit an Activity.
2. Begin typing in the "Who was involved?" text input to select a new person.
3. Tap the person from the autocomplete list. The tap has no effect.

**Comment by tbirrell** [comment]
For what its worth, this is not a problem on my Pixel (Android P, Self hosted, v2.14.0), it is not a mobile-wide issue.

Its been awhile since I've used an iPhone, but is there a way to detect tap events? I suspect that the tap is being intercepted by something else. What happens when you:

- Close keyboard and tap
- Search from dashboard and tap
- Search from dashboard, close keyboard, and tap

Is its the same issue no matter what, its a problem with the drop down. If closing the keyboard first helps, then the tap is getting intercepted. If using the menu on the dashboard works but this one still doesn't, then I'm not really sure, but hopefully the additional information helps someone else debug the problem.

**Comment by reid** [comment]
Thanks! Closing the keyboard and tapping works the first time.

When I try to add another person, the trick does _not_ work again.

I can close the keyboard by pressing the iOS "Done" button right above the keyboard.

**Comment by tbirrell** [comment]
So to be clear, you can now get two people added to an activity? The original individual and one more. But then you can't add a third?

**Comment by reid** [comment]
Correct. I can use the close-keyboard trick to select the 2nd person. But I can't add anyone else.
